### DISCUSSION TITLE
How to change language when I use the Hugging Face API?

### DISCUSSION THREAD

**Discussion by Zapzatron** [openai/whisper#1419] [Q&A]
Why the input language is Russian, it understands it and translates it into English and sends it to me.
Words are lost when translated into English.
Can I get speech recognition without translation? 

```python
import requests

API_URL = "https://api-inference.huggingface.co/models/openai/whisper-large-v2"
headers = {"Authorization": "Bearer hf_..."}

def query(filename):
    with open(filename, "rb") as f:
        data = f.read()
    response = requests.post(API_URL, headers=headers, data=data)
    return response.json()

output = query("sample1.flac")
```
https://huggingface.co/openai/whisper-large-v2
https://huggingface.co/openai/whisper-large-v2/discussions/47#6479171412ae749b62ec5223
https://discord.com/channels/879548962464493619/1113954842302500975

**Comment by a1ansa1a** [comment]
Did you find a solution?
